nibabel/freesurfer/io.py

```diff
@@ -296,7 +326,20 @@ def write_morph_data(file_like, values, fnum=0):
 
 
 def read_annot(filepath, orig_ids=False):
-    """Read in a Freesurfer annotation from a .annot file.
+    """Read in a Freesurfer annotation from a `.annot` file.
+
+    An `.annot` file contains a sequence of vertices with a label (also known
+    as an "annotation value") associated with each vertex, and then a sequence
+    of colours corresponding to each label.
+
+    The colour table itself may be stored in either an "old-style" format, or
+    a "new-style" format - the :func:`_read_annot_ctab_old_format` and
```

Comment:
Prefixing variables and functions with `_` indicates they're private, and I don't think the docstrings you're linking to will show up in the docs. (Also, referring to them in the docs might negate some of the point of making them private in the first place.)

Maybe, this paragraph could just be:

```
Annotation file format versions 1 and 2 are supported, corresponding to the
"old-style" and "new-style" color table layout.
```

This way, if a version 3 ever happens, the docs are less likely to become incorrect.